dist/images/ovnkube.sh

```diff
@@ -290,13 +290,14 @@ check_ovn_daemonset_version() {
 get_ovn_db_vars() {
   ovn_nbdb_str=""
   ovn_sbdb_str=""
-  for i in ${!ovn_db_hosts[@]}; do
-    if [[ ${i} -ne 0 ]]; then
+  for i in "${ovn_db_hosts[@]}"; do
```

Comment:
There are two more brackets on IPv4 addresses. Not sure if you care about those:
~Line 654:
  ovn-nbctl --inactivity-probe=0 set-connection p${transport}:${ovn_nb_port}:[${ovn_db_host}]
~Line 686
  ovn-sbctl --inactivity-probe=0 set-connection p${transport}:${ovn_sb_port}:[${ovn_db_host}]

Reply:
thanks I'll fix this